src/components/core/geoChart/geoChartDataSource.ts

```diff
@@ -29,6 +39,7 @@ function transformPushpinDataSource(geoData: IGeoData): IGeoDataSourceFeatures {
     const locationNameData = hasTooltip ? tooltipText.data : [];
     const segmentData = hasSegment ? segment.data : [];
     const sizeData = hasSize ? size.data : [];
+    const { min: minSizeFromData, max: maxSizeFromData } = getMinMax(sizeData);
```

Comment:
Please move this line into `calculatedSizeInPixel` function.
=> `calculatedSizeInPixel(sizeData[index], sizeData, geoPointsConfig)`
 :)

Reply:
i just want to calculate it outside the loop, and reuse it instead of call it again and again in loop `locationData.reduce`